core/controllers/admin_test.py

```diff
@@ -2275,3 +2277,27 @@ def test_flush_memory_cache(self):
         response = self.get_json(
             '/memorycacheadminhandler')
         self.assertEqual(response['total_keys_stored'], 0)
+
+
+class NumberOfDeletionRequestsHandlerTest(test_utils.GenericTestBase):
+    """Tests NumberOfDeletionRequestsHandler."""
+
+    def setUp(self):
+        super(NumberOfDeletionRequestsHandlerTest, self).setUp()
+        self.signup(self.ADMIN_EMAIL, self.ADMIN_USERNAME)
+        self.login(self.ADMIN_EMAIL, is_super_admin=True)
+
+    def test_get_with_no_deletion_request_returns_zero(self):
+        response = self.get_json('/numberofpendingdeletionrequests')
+        self.assertEqual(response['number_of_pending_deletion_models'], 0)
+
+    def test_get_with_two_deletion_request_returns_zero(self):
```

Comment:
"returns zero"...?

Please proofread/double-check your PR more carefully (names + comments + docstrings) in future.

Reply:
Done.